src/cards/f-cards.ts

```diff
@@ -7,8 +33,56 @@ addBasicCard('FINE MOTOR SKILLS', 'purple', 'Classic', 2);
 addBasicCard('FIRE SKIN', 'red', 'Classic', 3);
 addBasicCard('FLATULENCE', 'colourless', 'Classic', 3);
 addBasicCard('FLIGHT', 'blue', 'Classic', 2);
-addBasicCard('FLOURISH', 'green', 'Classic', );
-addBasicCard('FORTUNATE', 'green', 'Classic', 1);
-addBasicCard('FREE WILL', 'colourless', 'multi-colour', 2);
+addBasicCard('FLOURISH', 'green', 'Classic', 0);
+const fortunate: PlayerCard = {
+  name: 'FORTUNATE',
+  type: ['green'],
+  pack: 'Classic',
+  calcA: (inst: CardInstance): void => {
+    inst.finalA = 1;
+  },
+  calcB: (
+    inst: CardInstance,
+    allPlayerCards: Array<Array<CardInstance>>,
+    currentPlayer: number
+  ): void => {
+    const playerCards = allPlayerCards[currentPlayer];
+    let greenSize = 0
+    let colours: { [key: string]: number } = {}
+    playerCards.forEach(
+      (inst) => {
+        inst.type.forEach(type => {
+          if(type =='green') {
+            greenSize += 1
+          } else  {
+            colours[type] = colours[type] ?? 0
+            colours[type] += 1
+          }
+        })
+      }
+    );
+    let maxSize = Math.max(...Object.values(colours))
+    // only when more green than others
+    inst.finalB = maxSize < greenSize ? 2 : 0
```

Comment:
```suggestion
    inst.finalB = maxSize <= greenSize ? 2 : 0
```

Only when more green than others!